packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -156,6 +157,7 @@ const AppStack = createStackNavigator(
     [Screens.PaymentRequestListScreen]: { screen: PaymentRequestListScreen },
     [Screens.ReclaimPaymentConfirmationScreen]: { screen: ReclaimPaymentConfirmationScreen },
     [Screens.FeeEducation]: { screen: FeeEducation },
+    [Screens.FeeExchangeEducation]: { screen: FeeExchangeEducation },
```

Comment:
I believe this should go in the exchange stack

Reply:
The problem is that could work a bit strange if is reused on another part of the app, similar to `FeeEducation`. But is true that is not the case right now, so I'll change it.